src/lib/hash/sha2_64/sha2_64_armv8/sha2_64_armv8.cpp

```diff
@@ -14,7 +14,7 @@ namespace Botan {
 /*
 * SHA-512 using CPU instructions in ARMv8
 */
-BOTAN_FUNC_ISA("+crypto")
+BOTAN_FUNC_ISA("+crypto,sha3")
```

Comment:
@bazineta Could you instead please try this on your machine:

```suggestion
BOTAN_FUNC_ISA("arch=armv8.2-a+sha3")
```

That seems to work with the CI's qemu-based arm64 target.

Reply:
🎉 Hooray!

Reply:
Will revise this PR to contain that change.